homeassistant/components/philips_js/remote.py

```diff
@@ -0,0 +1,108 @@
+"""Remote control support for Apple TV."""
+
+import asyncio
+
+from haphilipsjs.typing import SystemType
+
+from homeassistant.components.remote import (
+    ATTR_DELAY_SECS,
+    ATTR_NUM_REPEATS,
+    DEFAULT_DELAY_SECS,
+    RemoteEntity,
+)
+
+from . import LOGGER, PhilipsTVDataUpdateCoordinator
+from .const import CONF_SYSTEM, DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the configuration entry."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities(
+        [
+            PhilipsTVRemote(
+                coordinator,
+                config_entry.data[CONF_SYSTEM],
+                config_entry.unique_id,
+            )
+        ]
+    )
+
+
+class PhilipsTVRemote(RemoteEntity):
+    """Device that sends commands."""
+
+    def __init__(
+        self,
+        coordinator: PhilipsTVDataUpdateCoordinator,
+        system: SystemType,
+        unique_id: str,
+    ):
+        """Initialize the Philips TV."""
+        self._tv = coordinator.api
+        self._coordinator = coordinator
+        self._system = system
+        self._unique_id = unique_id
+
+    @property
+    def name(self):
+        """Return the device name."""
+        return self._system["name"]
+
+    @property
+    def is_on(self):
+        """Return true if device is on."""
+        if self._tv.on:
+            if self._tv.powerstate == "On" or self._tv.powerstate is None:
```

Comment:
It's a bit unclear how to behave here. How should this model behave when the TV is in standby? It still accept sent commands in that case. So in some sense it is still on. I have currently matched the behaviour of the media_player entity, but not sure if that is correct here.

Reply:
best to be in sync with media player.